### DISCUSSION TITLE
Setting a background-image with a custom property.

### DISCUSSION THREAD

**Discussion by imade-nl** [tailwindlabs/tailwindcss#12882] [Ideas]
I would like to set a background-image trough a custom property:
css
```
:root {
  --rickroll: url('https://media.giphy.com/media/5kq0GCjHA8Rwc/giphy.gif');
}
```

html
```
<div class="h-96 w-96 bg-cover bg-center bg-img-[--rickroll]"></div>
```

**Comment by wongjn** [comment]
The class would be `bg-[image:--rickroll]`, see [this Tailwind Play](https://play.tailwindcss.com/IVpEl0OyOg).

**Reply by imade-nl** [reply]
Excellent, thank you so much! I've missed that in the docs.
